lib/Backends/NNPI/NNPIDeviceManager.cpp

```diff
@@ -415,14 +408,11 @@ NNPIStaticPlaceholderContainer::~NNPIStaticPlaceholderContainer() {
   }
 }
 
-bool NNPIStaticPlaceholderContainer::SetDevice(NNPIDeviceContext device,
-                                               bool inferOnRuntime) {
+bool NNPIStaticPlaceholderContainer::SetDevice(NNPIDeviceContext device) {
```

Comment:
nit: Just noticed this and the other functions in `NNPIStaticPlaceholderContainer` are not `lowerCamelCase`, would be good to cleanup at some point.